main.go

```diff
@@ -34,7 +34,7 @@ func main() {
 	}
 
 	stopChan := make(chan string)
-	go master(stopChan)
+	go master(&stopChan)
```

Comment:
BTW, I don't see any reason not to pass a channel pointer here, but what breaks if we don't pass a pointer? https://stackoverflow.com/a/16590291

cc @rogeruiz; I never understood what this change was for.